packages/shopify-codemod/transforms/remove-redundant-returns-from-addeventlistener.js

```diff
@@ -0,0 +1,28 @@
+export default function removeAddEventListener(file, api) {
```

Comment:
What are the thoughts on also having this transform remove any arguments to `return` statements nested in blocks within the function?